chainer_compiler/chainer_compiler.py

```diff
@@ -203,6 +203,34 @@ def _run_translator(translator, mc, inputs):
     return graph
 
 
+def _resolve_name_correspondence(mc, params, name, param_values):
+    for running_mean_name in ('avg_mean', 'running_mean'):
+        if not name.endswith(running_mean_name):
+            continue
+        parent = name[:-len(running_mean_name)]
+        mean = params[parent + 'beta']
+        for link in mc.links():
```

Comment:
I think iterating over `mc.links` should be done around line 300, before `for name in self.param_names:`.